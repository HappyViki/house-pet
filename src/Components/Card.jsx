import React from 'react'

function Card (props) {
  console.log('Card', props)
  return (
    <div className='card'>
      <img src={props.src} />
      <p>Distance: {props.distance} miles away</p>
      <a href={props.url}>Go to pet on Petfinder.com</a>
    </div>
  )
}

export default Card
