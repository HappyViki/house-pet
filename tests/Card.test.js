import React from 'react'
import Card from '../src/Components/Card.jsx'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer
    .create(<Card key='1' id='687845' src='image.png' distance='6.254' url='website.com' />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
