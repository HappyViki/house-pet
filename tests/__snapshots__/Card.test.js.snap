// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="card"
>
  <img
    src="image.png"
  />
  <p>
    Distance: 
    6.254
     miles away
  </p>
  <a
    href="website.com"
  >
    Go to pet on Petfinder.com
  </a>
</div>
`;
